docs/tutorials/services/webcam-line-follower-robot.md

```diff
@@ -85,103 +170,96 @@ Or if you prefer the raw JSON:
+But first, you need to get creative and use your colored tape to make a path for your robot to follow.
+Perhaps a circle or other shape, or perhaps a path from one point of interest to another.
+Sharp corners will be more challenging for the robot to follow so consider creating more gentle curves.
 
-{{% /tab %}}
-{{< /tabs >}}
+Once you have created your path, set your robot on the line such that the line appears in the front of the camera’s view.
+Verify that the camera sees the line by viewing the camera feed on the **Control** tab of the robot page.
 
-### Configuring the visualizer
+{{<imgproc src="/tutorials/webcam-line-follower/lf-cam-view6.png" resize="600x" declaredimensions=true alt="The camera view in the control tab on the robot page">}}
 
-This step is optional, but if you'd like to see the bounding boxes that the color detector identifies, you'll need to configure a [transform camera](/components/camera/transform/).
-This isn't another piece of hardware, but rather a virtual "camera" that takes in the stream from the webcam we just configured and outputs a stream overlaid with bounding boxes representing the color detections.
+Now, let's configure the color detector so your rover can detect the line:
 
-In the **Config** tab, make a new component with name `show_detections`, type `camera` and model `transform`.
-Set the `stream` to `"color"` and set the `source` to `"my_camera"` or whatever you named your webcam.
+{{< tabs >}}
+{{% tab name="Builder UI" %}}
 
-You'll need to edit the `pipeline` section as well with `type` set to `"detections"`, and `detector_name` set to the name of your color detector (`"green_detector"` in our case).
+Next, navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Services** subtab and navigate to the **Create service** menu.
 
-You can paste the following into the **Attributes** section of the `show_detections` config builder:
+1. **Add a vision service.**
 
-```json {class="line-numbers linkable-line-numbers"}
-{
-  "stream": "color",
-  "source": "my_camera",
-  "pipeline": [
-    {
-      "type": "detections",
-      "attributes": {
-        "detector_name": "green_detector"
-      }
-    }
-  ]
-}
-```
+   Next, add a [detector](/services/vision/detection/) as a vision service to be able to make use of the ML model.
+   Create an vision service with the name `green_detector`, the type `vision` and the model `color_detector`.
+   Then click **Create Service**.
 
-If you save the config and go to the **Control** tab, you should now be able to view the camera feed with color detector overlays superimposed on the image.
+   In your vision service’s panel, select the color your vision service will be detecting, as well as a hue tolerance and a segment size (in pixels).
+   Use a color picker like [colorpicker.me](https://colorpicker.me/) to approximate the color of your line and get the corresponding hexadecimal hash to put in your config.
```

Comment:
nit: most refer to this as the `hex code`, but `hexadecimal code` is fine if you want more formal. `Hash` is more for secrets, salt, and other uniquifiers, oh my.

```suggestion
   Use a color picker like [colorpicker.me](https://colorpicker.me/) to approximate the color of your line and get the corresponding hex code to put in your config.
```